cmake/Modules/Packages/LATTE.cmake

```diff
@@ -18,17 +20,20 @@ if(PKG_LATTE)
       -DBLAS_LIBRARIES=${BLAS_LIBRARIES} -DLAPACK_LIBRARIES=${LAPACK_LIBRARIES}
```

Comment:
I guess set should look up the location here, instead of doing the hack above, something like:
```
get_target_property(BLAS_LOCATION linalg LOCATION)
```

Reply:
this gives an error.